quarkus.ls/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/services/QuarkusJavaHover.java

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+* Copyright (c) 2019 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package com.redhat.quarkus.services;
+
+import com.redhat.quarkus.commons.QuarkusJavaHoverInfo;
+import com.redhat.quarkus.utils.DocumentationUtils;
+
+import org.eclipse.lsp4j.Hover;
+import org.eclipse.lsp4j.MarkupContent;
+
+/**
+ * Quarkus Java hover.
+ *
+ */
+public class QuarkusJavaHover {
+
+	public static Hover doHover(QuarkusJavaHoverInfo hoverInfo, boolean markdownSupported) {
```

Comment:
The services package is for application.properties. Move this method in DocumentationUtils